.github/CONTRIBUTING.md

```diff
@@ -10,6 +10,23 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
 
 ### Propose a feature
 
-* Please create a pull request with your change proposal in the [Homebrew Evolution](https://github.com/Homebrew/brew-evolution) repository using the [proposal template](https://github.com/Homebrew/brew-evolution/blob/master/proposal_template.md).
+* create a pull request in the [Homebrew Evolution](https://github.com/Homebrew/brew-evolution) repository using the [proposal template](https://github.com/Homebrew/brew-evolution/blob/master/proposal_template.md)
+
+### Contribute code
```

Comment:
I'm not convinced we need this section, really. It's too generic to be particularly useful unless it's just a single bullet pointing to https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged